In [1]:
import pandas as pd
import numpy as np

df = pd.read_csv("data/results.csv")

In [6]:
print("Mean A: " + str(np.mean(df["A_error"])))
print("Mean B: " + str(np.mean(df["B_error"])))

Mean A: 1.55
Mean B: 3.0


In [ ]:
# Pearson correlation
df[["A_error", "B_error"]].corr()

A_error   B_error
A_error  1.000000  0.779101
B_error  0.779101  1.000000

In [ ]:
# Spearman correlation

df[["A_error", "B_error"]].corr(method="spearman")

A_error  B_error
A_error   1.0000   0.7375
B_error   0.7375   1.0000

Means: Mr A. makes guesses almost twice closer to the true age than Mrs B. On average A overestimates age by 1.55 year 
and B by 3 years.

Correlations: there is high correlation between A and B's errors, which indicates their similar guesses.